BusinessLogic/Entities/Subscription.cs

```diff
@@ -99,7 +100,13 @@ public async Task<HttpResponseMessage> SendEvent(Event _event)
                 {
                     foreach (var callback in CallBacks)
                     {
+                        if (this.Synchronous)
```

Comment:
Add a comment above:

```suggestion
                        // if the subscription is synchronous then it should only have one callback so we're safe returning its response here
                        if (this.Synchronous)
```

Reply:
fixed too